xbps-mini-builder

```diff
@@ -4,9 +4,9 @@
 cd "${0%/*}" || exit 1
 
 # Do we have keys to sign with?
-if [ ! -f id_rsa ] ; then
-    rm -rf id_rsa id_rsa.pub
-    ssh-keygen -b 4096 -t rsa -N "" -f id_rsa
+if [ ! -f id_rsa.pem ] ; then
```

Comment:
Changing this filename breaks backwards compatibility and will cause existing users to have their entire repos untrusted.  Please keep the same filename and only modify the options.